packages/fastify-siwe/src/registerSiweRoutes.ts

```diff
@@ -40,19 +41,20 @@ export const registerSiweRoutes = (
       if (!signature || !message) {
         return reply.status(422).send({ message: 'Expected prepareMessage object and signature as body.' })
       }
-      const token = JSON.stringify({ signature, message })
+      const { address, chainId } = message
+      const token = { signature, message } as Token
```

Comment:
```suggestion
      const token: Token = { signature, message }
```

Can you do this? Will be a little safer